### DISCUSSION TITLE
Xml Namespace for Scheduler Configuration File

### DISCUSSION THREAD

**Discussion by zoldbogar** [quartznet/quartznet#2300] [Q&A]
Hello,
Our code was using `http://quartznet.sourceforge.net/JobSchedulingData` as xmlns for the scheduler config file, but I've noticed that this should be updated to `http://www.quartz-scheduler.org/xml/JobSchedulingData`. My problem is that neither of above mentioned files exist. Is there a newer version available somewhere?

Thanks,
zb

**Comment by jafin** [comment]
The namespace is just that a namespace by convention.  It doesn't need a physical entity to exist at the end of the url.

If you look here the namespace `http://quartznet.sourceforge.net/JobSchedulingData` is used in the attributes.

https://github.com/quartznet/quartznet/blob/fe5420a8b206fd4e3954e9921146d42a05c3f345/src/Quartz/Xml/job_scheduling_data_2_0.cs#L345

There is an example that has a sample XML, is that what you are seeking?
https://github.com/quartznet/quartznet/blob/fe5420a8b206fd4e3954e9921146d42a05c3f345/src/Quartz.Examples/quartz_jobs.xml#L4

**Reply by zoldbogar** [reply]
Thanks for the reply.

Actually VisualStudio is trying to validate the XML schema, but in lack of an `xmlns` document it shows messages like 
`Could not find schema information for the element 'http://www.quartz-scheduler.org/xml/JobSchedulingData:schedule'.`

Adding `xmlns:xsi` doesn't help either.
